### PR TITLE
Show a warning when the vanilla launcher is open.

### PR DESCRIPTION
**Pull request by modmuss50**
https://user-images.githubusercontent.com/4324090/118359529-c07db200-b57b-11eb-8904-628fb13679fe.mp4

Needs further testing and testing on other arches. Currenly windows only, mac os is mostly done just a slight issue on the native build side of things.

### CODE REVIEW

File: src/main/java/net/fabricmc/installer/launcher/NativesHelper.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        natives.put("windows-arm64", "natives/windows-ARM64.dll");
+        natives.put("windows-Win32", "natives/windows-Win32.dll");
+        natives.put("windows-amd64", "natives/windows-x64.dll");
+
+        return natives;
+    }
+
+    private static String getOS() {
+        String osName = System.getProperty("os.name").toLowerCase();
```
Comment (sfPlayer1) [inline src/main/java/net/fabricmc/installer/launcher/NativesHelper.java:49]:
```suggestion
        String osName = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
```

Needs the proper import ofc

**Review by sfPlayer1 — COMMENTED** [review]
Linux and other *nix should be possible in plain Java by testing `/tmp/minecraftlauncher.<uid>.pid.lock` (per [1] - not tested). The uid is obtainable by executing `id -u`. Maybe osx uses the same file?


[1] https://discord.com/channels/507304429255393322/566418023372816394/840716250497417218

**Comment by jamierocks** [comment]
Why have the JNI implementation in a separate repository?

**Comment by modmuss50** [comment]
Makes it much easier for most people to build this repo. Means we don’t need to do any cross compile funkiness either, just pull the natives from maven.

**Review by sfPlayer1 — APPROVED** [review]

**Pull request merged**